generator/src/main/java/com/scottlogic/deg/generator/validators/ValidationResult.java

```diff
@@ -12,4 +12,8 @@ public ValidationResult(ArrayList<String> errorMessages){
     public boolean isValid() {
         return errorMessages.isEmpty();
     }
+
+    public void displayErrors() {
+        errorMessages.forEach(e -> System.err.println("* " + e));
```

Comment:
Why has the reporter been removed? The result should be a collection of results and the reporter responsible for displaying them in the appropriate manner, or not at all if so desired.